modules/con_ipi-installation-workflow.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-prerequisites.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="installation-workflow_{context}"]
+= Installation workflow
+
+The installation program supports interactive mode, but it is recommended to prepare an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts.
+
+The administrator adds the relevant cluster details to the `install-config.yaml` file. The `install-config.yaml` file is shipped to the provisioning host. The administrator generates the manifests and verifies all prerequisites.
+
+The installation program enrolls all nodes in the cluster and starts the bootstrap VM. Metal platform components are started as `systemd` services, which have the following containers:
+
+* Ironic-dnsmasq: DHCP server responsible for handing over the IP addresses to the provisioning interface of various nodes on the provisioning network. Ironic-dnsmasq is only enabled when the `provisioningNetwork` is enabled.
+* Ironic-httpd: http server used to ship the images to the nodes.
+* Image-customization
+* Ironic
+* Ironic-inspector (available in {openshift} 4.16 and earlier)
+* Ironic-ramdisk-logs
+* Extract-machine-os
+* Provisioning-interface: Service that configures networking when the provisioning interface is enabled.
+* Metal3-baremetal-operator: Operator that provides a higher-level API for Ironic and replaces `terraform-provider-ironic` (available in {openshift} 4.16 and later)
+
+The nodes enter the validation phase. The nodes move to a “manageable” state after Ironic validates the credentials to access the Baseboard Management Controller (BMC) for each node.
+
+Once in the “manageable” state, the “inspection” phase starts. The "inspection" phase ensures that the hardware meets the minimum requirements needed for a successful deployment of {ocp}.
+
+The `install-config.yaml` file details the provisioning network. The installation program on the bootstrap VM will try to push a live image to every node with the Ironic Python Agent (IPA) loaded in the RAM. When using virtual media, the live image connects directly to the BMC.
```

Comment:
I don't think you need to say "in the RAM" here. Check on that last clause with QE. I would think the BMC connects with the live image, since the image is just an ISO.

Reply:
Updated as suggested. Will check with QE on the second point.

Reply:
@sgoveas, could you please confirm if the last clause on line 30 is correct?